docs/src/man/basics.md

```diff
@@ -296,8 +303,8 @@ In this way we can always easily restore our data even if we mess up the
 
 ### Basic Operations on Data Frames
```

Comment:
the sentence below is changed

Reply:
Couldn't "reading" actually confuse people? Once the column has been extracted it can be written to and that mutates the data frame.